src/main/cpp/.clang-format

```diff
@@ -0,0 +1,155 @@
+---
```

Comment:
it looks like this is merely an exact copy of  cudf's `.clang-format`. We should use the original file to stay in sync if possible

Can you try adding 
```
-style='file:thirdparty/cudf/cpp/.clang-format' 
```
and remove the copy if it works?